templates/artist/artist_detail.html

```diff
@@ -24,23 +24,26 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <div class="break">
         <hr>
     </div>
-    {% thumbnail artist.photo.img "500" as thumb %}
-        <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
-    {% endthumbnail %}
+    <div class="extra-padding">
+        {% thumbnail artist.photo.img "500" as thumb %}
+            <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
```

Comment:
I think you might be able to apply the `extra-padding` class to the `<img>` tag and skip the use of `<div>` altogether. Do you mind giving that a try?